ui/src/router/components/form/components/VersionCreationSummary.js

```diff
@@ -0,0 +1,16 @@
+import React, { Fragment } from "react";
+import { EuiSpacer } from "@elastic/eui";
+
+export const VersionCreationSummary = ({ router }) => {
+  return (
+    <Fragment>
+      <p>
+        You're about to create a new version for the Turing router{" "}
+        <b>{router.name}</b> in the <b>{router.environment_name}</b> environment
+        without deploying it.
```

Comment:
Hmm that's true. I think I based that on what was written for the deployment confirmation (in `DeploymentSummary.js`):
```
You're about to deploy a new Turing router <b>{router.name}</b> into{" "}
<b>{router.environment_name}</b> environment.
```
which I guess, going by the same logic, shouldn't mention the environment either?

But yeah I'll update it to make it clearer and less... excessive in providing random information.

Reply:
Okay, I guess because `DeploymentSummary.js` is currently being used to display messages for both Router creation and update, `router.environment_name` is being mentioned in the confirmation summary for both use cases. I guess I can sacrifice a bit of code reuse here by splitting this into 2 files, `CreationSummary.js`, which is going to carry the existing message, and `UpdateSummary.js`, which will remove mention of the environment, and emphasise the deployment of the new version.